addons/common/functions/fnc_statusEffect_set.sqf

```diff
@@ -32,7 +32,7 @@ if (isNull _object) exitWith {TRACE_1("null",_object);};
 [_object, true] call FUNC(statusEffect_resetVariables); //Check for mismatch, and set object ref
 
 //check ID case and set globally if not already set:
-_ID = toLower _ID;
+_ID = toLowerANSI _ID;
```

Comment:
I don't think it's a good idea here either.

Reply:
https://community.bistudio.com/wiki/find only supports ANSI characters unless forceUnicode'd